<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1912</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblHederName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1900</width>
      <height>77</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblCommStatus">
    <property name="geometry">
     <rect>
      <x>1479</x>
      <y>10</y>
      <width>181</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0); 
color:white;
border-bottom-right-radius:10px;
border-top-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>EETIS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblDateTime">
    <property name="geometry">
     <rect>
      <x>1668</x>
      <y>10</y>
      <width>230</width>
      <height>52</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,60%);
color:white;
border-bottom-right-radius:10px;
border-top-left-radius:10px;</string>
    </property>
    <property name="text">
     <string>18-01-2025 | 12:12:12</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>773</x>
      <y>22</y>
      <width>350</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>EETIS_TESTJIG_SIMULATOR</string>
    </property>
   </widget>
   <widget class="QFrame" name="frmMain">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>90</y>
      <width>1900</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>998</y>
      <width>1900</width>
      <height>82</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:color rgb(150,160,186,40%);
border-radius:10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pbBDOBUS">
     <property name="geometry">
      <rect>
       <x>354</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:00px;
border-top-right-radius:00px;

border-bottom-left-radius:0px;
border-top-left-radius:00px;</string>
     </property>
     <property name="text">
      <string>BDOBUS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBfcmdf">
     <property name="geometry">
      <rect>
       <x>609</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:00px;
border-top-right-radius:00px;

border-bottom-left-radius:0px;
border-top-left-radius:00px;</string>
     </property>
     <property name="text">
      <string>BFCMDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbStdp">
     <property name="geometry">
      <rect>
       <x>864</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:00px;
border-top-right-radius:00px;

border-bottom-left-radius:0px;
border-top-left-radius:00px;</string>
     </property>
     <property name="text">
      <string>STDP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>1119</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:00px;
border-top-right-radius:00px;

border-bottom-left-radius:0px;
border-top-left-radius:00px;</string>
     </property>
     <property name="text">
      <string>FCU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>1374</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:00px;
border-top-right-radius:00px;

border-bottom-left-radius:0px;
border-top-left-radius:00px;</string>
     </property>
     <property name="text">
      <string>CCU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRfu">
     <property name="geometry">
      <rect>
       <x>1629</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:10px;
border-top-right-radius:10px;

border-bottom-left-radius:0px;
border-top-left-radius:00px;</string>
     </property>
     <property name="text">
      <string>RFU</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbBBAT">
     <property name="geometry">
      <rect>
       <x>99</x>
       <y>15</y>
       <width>255</width>
       <height>55</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(53, 74, 131); 
color: rgb(255, 255, 255);
border-bottom-right-radius:00px;
border-top-right-radius:00px;

border-bottom-left-radius:10px;
border-top-left-radius:10px;</string>
     </property>
     <property name="text">
      <string>BBAT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShutDown">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>1</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>26</horstretch>
       <verstretch>18</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(234, 236, 247); 

color:white;
border-radius:50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lblDntLogo">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>8</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(53, 74, 131); 
background-image: url(:/images/DNT_Logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
